### ISSUE TITLE
sitewide, design system, components [KEYBOARD]: Focused item MUST be contained by focus outline, pagination Next >

### ISSUE THREAD

**Issue by jenstrickland** [department-of-veterans-affairs/va.gov-team#8255]
# [508-defect-3](https://github.com/department-of-veterans-affairs/va.gov-team/blob/master/platform/accessibility/guidance/defect-severity-rubric.md#508-defect-3)

```diff
! Team: VSA Public Websites; Project: Find Forms MVP
```

**Feedback framework**

- **❗️ Must** for if the feedback must be applied 
- **⚠️Should** if the feedback is best practice 
- **✔️ Consider** for suggestions/enhancements 

## Description

When focus is on the last item in the pagination the `>` is outside of the focus outline, but should be within.

## Point of Contact

**VFS Point of Contact:** Jennifer

## Acceptance Criteria

As a keyboard user, I want to tab through the page and identify where focus is.

## Environment

* Operating System: all
* Browser: all
* Server destination: all

## Steps to Recreate

1. Enter https://www.va.gov/find-forms?q=medical in browser
2. Tab through the content, to the last item in the pagination
3. Verify that the `>` is not enclosed by the focus outline

## Screenshots or Trace Logs

![Screen Shot 2020-04-21 at 8 32 10 PM](https://user-images.githubusercontent.com/57469/79928109-f2bbef00-840f-11ea-8d0f-af76f3a78aad.png)

**Comment by 1Copenut** [comment]
Issue moved to [department-of-veterans-affairs/vets-design-system-documentation #248](https://github.com/department-of-veterans-affairs/vets-design-system-documentation/issues/248) via [**ZenHub**](https://www.zenhub.com/)

**Comment by jenstrickland** [comment]
As this bug is still in the VSA Public Website's work, we should keep this bug open. Once the sitewide issue is resolved, we should ensure that the update is reflected in this work.

**Comment by MarciMcGuireGCIO** [comment]
Moving this out of https://github.com/department-of-veterans-affairs/va.gov-team/issues/8694 and into the https://github.com/department-of-veterans-affairs/va.gov-team/issues/16897 epic so that the Staging Review epic can be closed once everything that is not sitewide is addressed.

**Comment by caw310** [comment]
Closing as this is old and the VADS components have been updated so this should no longer be an issue. IF you are still seeing the issue with the new components, file a new bug.
